app/src/main/java/com/teamfillin/fillin/presentation/map/StudioMapActivity.kt

```diff
@@ -0,0 +1,123 @@
+package com.teamfillin.fillin.presentation.map
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import com.naver.maps.geometry.LatLng
+import com.naver.maps.map.*
+import com.naver.maps.map.util.FusedLocationSource
+import com.teamfillin.fillin.databinding.ActivityStudioMapBinding
+
+
+class StudioMapActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityStudioMapBinding
+    private lateinit var locationSource: FusedLocationSource
+    private var naverMap: NaverMap? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityStudioMapBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.mapMain.onCreate(savedInstanceState)
+        locationSource =
+            FusedLocationSource(this, LOCATION_PERMISSION_REQUEST_CODE)
+        binding.mapMain.getMapAsync(NaverMapProvider(locationSource, naverMap, binding))
+        toolbarEvent()
+        searchClickEvent()
+
+    }
+
+    private fun toolbarEvent() {
+        setSupportActionBar(binding.toolbar)
+        val ab = supportActionBar!!
+        ab.setDisplayHomeAsUpEnabled(true)
+        ab.setDisplayShowTitleEnabled(false)
```

Comment:
변수명이 추상적인 것 같아! ab라고만하면 어떤 의미인지 잘 이해가 안될 것 같은 느낌

```suggestion
        supportActionBar?.run {
            setDisplayHomeAsUpEnabled(true)
            setDisplayShowTitleEnabled(false)
        }
```

그리고 위와 같이 임시변수는 필요없이 바로 달리면 됩니당

Reply:
👍